### ISSUE TITLE
Customizer: Errors in Console: "Failed to execute 'postMessage' on 'DOMWindow'"

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-calypso#6357]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com
2. Choose a site and load the customizer

#### What I expected

No console errors

#### What happened instead

Multiple JavaScript console errors: 

`https://s2.wp.com/wp-includes/js/customize-base.min.js?m=1461081265h&ver=4.5.3 1:6832 Failed to execute 'postMessage' on 'DOMWindow': The target origin provided ('https://e2eflowtestingbackup.wordpress.com') does not match the recipient window's origin ('https://wordpress.com').`

#### Browser / OS version

Chrome/Firefox latest OSX 10.11.5

#### Screenshot / Video

![screen shot 2016-06-28 at 4 09 40 pm](https://cloud.githubusercontent.com/assets/128826/16405412/c7e104ce-3d4a-11e6-9af8-35a19116df3a.png)

#### Context / Source

#dogfooding

**Comment by sirbrillig** [comment]
This is a known issue, and is documented in the pre-OSS calypso repo (issue 9893), but it's a good idea to document it here too. Thanks for bringing it up! To quote myself:

The issue is: the customizer is running in an iframe inside Calypso, but in Core the only time the Customizer is in an iframe is in wp-admin, and when various actions happen Core tries to tell the parent page about them, except it's not wp-admin, it's Calypso, so you get a x-domain error. We would need to update WordPress core to prevent those errors.

TL;DR: this is a WordPress core issue and we can't fix it right now, but if we end up with a native calypso customizer, the problem will go away. 😄
